### PR TITLE
Add live data section to getting started guide

### PR DESCRIPTION
**Pull request by Daniel-Nielsen**

### CODE REVIEW

File: src/ios/v3/getting-started/livedata.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+eleventyNavigation:
+  title: Enable Live Data
+  key: ios-v3-getting-started-enable-live-data
+  parent: ios-v3-getting-started
+  order: 170
+---
+
+{% include "src/shared/live-data/live-data-intro.md" %}
+
+Enabling of Live Data through `MPMapControl` can be done as simple as calling `MPMapControl.enableLiveData()` with a [Domain Type]({{ site.url }}/ios/v3/guides/live-data/#domain-type).
```
Comment (simonlaustsen) [inline src/ios/v3/getting-started/livedata.md:13]: Should we have a list of all supported domain types or somehow reference to it?

**Comment by simonlaustsen** [comment]
A single comment otherwise fine 👍

**Review by timMikkelsen — APPROVED** [review]
Looks good!

File: src/ios/v3/getting-started/livedata.md (hunk 1 above)
Comment (Daniel-Nielsen) [reply src/ios/v3/getting-started/livedata.md:13]: Good point, I have created a jira for that work.

**Comment by ChristianWolf-mapspeople** [comment]
The text snippets extracted to `shared` should be used in the Android and web docs as well.

**Review by ChristianWolf-mapspeople — APPROVED** [review]

**Pull request merged**